docs/guides/cortex-with-cassandra.md

```diff
@@ -0,0 +1,174 @@
+## Running cortex with cassandra
+
+
+Cortex works as a backend store for prometheus that offers long term storage, multi-tenancy & global aggregation of metrics. 
+
+We can run cortex as a single binary either by running it as an container or as a process by building it from scratch.
+
+Now we are using cassandra as backend storage. Let's create a new cassandra instance by running the below commands or you can use the existing cassandra setup:
+
+Run cassandra with following cmd
+
+```
+docker run --name cassandra --rm -p 9042:9042 cassandra:3.11
+```
+
+ssh into cassandra container 
+
+```
+docker exec -it <container_id> bash
```

Comment:
What about directly use `docker exec -it <container_id> cqlsh`

Reply:
yeah. I will do that :) Thanks!